tests/e2e/functional_test.go

```diff
@@ -185,6 +186,12 @@ var (
 	monovertexgvr schema.GroupVersionResource
 )
 
+func TestE2E(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	RunSpecs(t, "E2E Suite")
+}
```

Comment:
I wonder if we'll want to call this `TestFunctionalE2E()` or `FunctionalTestE2E()`? and `RunSpecs(t, "Functional E2E Suite")`?